### PR TITLE
feat(config): add missing param and metadata for 46203

### PR DESCRIPTION
**Pull request by crxporter**
Copied missing data from this same device with different mfg/product IDs:

https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x0039/39351_jasco_zw3010.json

### CODE REVIEW

**Comment by AlCalzone** [comment]
@blhoward2 any concerns?

File: packages/config/config/devices/0x0063/46203.json [hunk 1]
```diff
@@ -33,28 +33,144 @@
 	},
 	"paramInformation": {
 		"3": {
-			"label": "LED Status",
+			"label": "LED Light",
+			"description": "LED status when device on or off.",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0063/46203.json]: Unnecessary description. Usually we say "LED Indicator."

File: packages/config/config/devices/0x0063/46203.json [hunk 2]
```diff
@@ -33,28 +33,144 @@
...
+				{
+					"label": "LED always ON",
+					"value": 3
+				}
+			]
 		},
 		"4": {
-			"label": "Invert",
+			"label": "Invert Switch",
+			"description": "Switch orientation.",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0063/46203.json]: Unnecessary description.

**Review by blhoward2 — CHANGES_REQUESTED** [review]
Minor nits.

**Comment by crxporter** [comment]
Give me a thumbs up on this comment if you'd like me to make those changes on all of these devices. It's at least 3, I can probably check some others I know are similar though. 

Thumbs down if you just want me to fix the single file in this PR.

**Comment by blhoward2** [comment]
You can, but this folder is on my list to template and go through anyway so it isn't necessary unless you really want to do so.

**Comment by crxporter** [comment]
Alright - for now I'll just stick with this file. I've made the requested edits.

I don't have any reason to get in the way of your process.

Thanks!

**Comment by crxporter** [comment]
Bump. I'm at a point I need these config options, can it be merged?

**Review by blhoward2 — APPROVED** [review]
Sorry, been busy. LGTM

**Pull request merged**